Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -99,6 +99,49 @@ class PaymentSheetUITest: XCTestCase {
         okButton.tap()
     }
 
+    func testPaymentSheetCustomDeferred() throws {
```

Comment:
how is this different from the other existing `testPaymentSheetCustomSaveAndRemoveCard_DeferredIntent_ServerSideConfirmation()`?  I think it's the fact that it updates, is there anything else?  We should rename this test to indicate what it's testing.  `testPaymentSheetCustomDeferred_update`?

Could we test the preservation of customer input, since we're testing the update flow?  Like, enter card details -> change the quantity -> expect card details to be preserved.

I'm not even sure we need to pay, if that part of the flow is covered by another test.

Reply:
Added a part of the test which tests input preservation, it passes since it doesn't use a leading zero expiry month.